src/components/Navbar.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import actions from '../actions/index';
+import CategoryFilter from './CategoryFilter';
+
+const Navbar = ({ changeFilter }) => {
+  const handleFilterChange = e => {
+    changeFilter(e.target.value);
+  };
+  return (
+    <header className="bgWhite">
+      <nav className="navbar" id="myTopnav">
+        <a className="navbarBrand azure" href="/">
+          Book Store
+        </a>
+        <nav>
+          <a href="#home" className="navItem books tt_u">
+            Books
+          </a>
+          <CategoryFilter filterHandler={handleFilterChange} />
+        </nav>
+        <nav className="mlAuto">
+          <a href="/avatar" className="avatar  navItem">
+            <img src="#" width="15" height="15" alt="A" />
```

Comment:
Why not make this a blue circle with div and "A" in it as white text, makes more sense that a broken image, or alternatively add an image.

Reply:
I'll go with adding a blue div with "A".